METIS/METIS_DET_IMG_N_GeoSnap.yaml

```diff
@@ -9,6 +9,8 @@ properties:
     temperature: -240
     dit: "!OBS.dit"
     ndit: "!OBS.ndit"
+    mindit: 0.011             # seconds, E-REP-NOVA-MET-1191, v2-0
+    full_well: !!float 2.8e6  # electrons, E-REP-NOVA-MET-1191, v2-0
```

Comment:
For the GeoSnap we also have a high capacity mode and a low capacity mode; see our hackathon minutes. [https://docs.google.com/document/d/1XLClrK8YGed4pQHLds3Q0T-LQHBLf4K4uW-mVcvR-yM/edit#heading=h.5a4s7xsi6p5y](url)

Reply:
Is it documented anywhere which detector mode is to be used for which observing mode? Do we expect a one-to-one relation? Not sure how to implement this - simplest would be to have to yaml files, one for high-, one for low-capacity mode. That would cause redundancies in the other parameters, though.

Reply:
No, there is no "final" concept for this yet. What I have done for now in the simulations is simply to do a check of the signal levels [e-/s/pix], and if they are lower than some fraction of the "detection capacity" (defined as well x frame rate, units [e-/s/pix]) then I assume the LCM is used, if the signal level supersedes said limit then HCM is used. In reality I think LCM will always be used for spectroscopy (but perhaps we need to allow HCM, too, in order to be able to observe very bright sources, TBC). And for imaging typically HCM will be used, for broadband imaging always for sure, though for the narrow band filters it may be beneficial to use LCM (TBC) but it depends on the observing conditions, what the real background will be, and on the science application. So here there is no simple answer unfortunately.